float32 vx
float32 vy
float32 az
float32 gimbal_forw